ansible_collections/arista/cvp/plugins/modules/cv_tag_v3.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+#
+# GNU General Public License v3.0+
+#
+# Copyright 2019 Arista Networks AS-EMEA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Required by Ansible and CVP
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: cv_tag_v3
+version_added: "4.0.0"
+author: PM Team (@aristanetworks)
+short_description: Create/Assign/Delete/Unassign tags on CVP
+description:
+  - CloudVison Portal Tag module to Create/Assign/Delete/Unassign tags on CloudVision
+options:
+  tags:
+    description: CVP tags
+    required: True
+    type: dict
+    elements: str
+  mode:
+    description: action to carry out on the tags
+                 create - create tags
+                 delete - delete tags
+                 assign - assign existing tags to device
+                 unassign - unassign existing tags from device
+    required: false
+    type: str
+    choices:
+      - create
+      - delete
+      - assign
+      - unassign
+  auto_create:
+    description: auto_create tags before assigning
+    required: false
+    default: true
+    type: bool
+'''
+
+EXAMPLES = '''
+---
+- name: "create tags"
+  arista.cvp.cv_tag_v3:
+    tags: "{{CVP_TAGS}}"
+    mode: create
+    auto_create: true
+'''
+
+import logging
+import traceback
+import ansible_collections.arista.cvp.plugins.module_utils.logger   # noqa # pylint: disable=unused-import
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.arista.cvp.plugins.module_utils.response import CvAnsibleResponse
+from ansible_collections.arista.cvp.plugins.module_utils import tools_cv
+from ansible_collections.arista.cvp.plugins.module_utils import tools_schema
+from ansible_collections.arista.cvp.plugins.module_utils.tag_tools import CvTagTools, CvTagInput
+try:
+    from cvprac.cvp_client_errors import CvpClientError, CvpApiError, CvpRequestError  # noqa # pylint: disable=unused-import
+    HAS_CVPRAC = True
+except ImportError:
+    HAS_CVPRAC = False
+    CVPRAC_IMP_ERR = traceback.format_exc()
+
+
+# Logger startup
+MODULE_LOGGER = logging.getLogger('arista.cvp.cv_tag_v3')
```

Comment:
```suggestion
MODULE_LOGGER = logging.getLogger(__name__)
```
As per Python documentation, avoid to hardcode the module name here.
https://docs.python.org/3/library/logging.html#logger-objects